<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_Log</class>
 <widget class="QDialog" name="Dialog_Log">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>460</width>
     <height>460</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="images.qrc">:/image/images/登录背景.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>401</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文行楷</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>欢迎使用学生成绩管理系统！</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_leave">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>410</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_help">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>410</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>帮助</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>180</y>
     <width>186</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>登录模式(&amp;M)</string>
        </property>
        <property name="buddy">
         <cstring>comboBox</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox">
        <item>
         <property name="text">
          <string>学生</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>教师</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>管理员</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>账号(&amp;A)</string>
        </property>
        <property name="buddy">
         <cstring>lineedit_account</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineedit_account"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>密码(&amp;P)</string>
        </property>
        <property name="buddy">
         <cstring>lineedit_password</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineedit_password">
        <property name="font">
         <font>
          <family>Agency FB</family>
          <pointsize>6</pointsize>
         </font>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_log">
      <property name="text">
       <string>登录</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>comboBox</tabstop>
  <tabstop>lineedit_account</tabstop>
  <tabstop>lineedit_password</tabstop>
  <tabstop>pushButton_log</tabstop>
  <tabstop>pushButton_help</tabstop>
  <tabstop>pushButton_leave</tabstop>
 </tabstops>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
